tslint.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
`tslint.json` is for only tests, so please put this file to `/test/types` directory.